tests/unit_tests/common/test_ldg_ptr.cu

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
I guess you thought about it, but probably there is not easy way to check that the feature is working.

Reply:
Indeed, we would have to disassemble the binary files or check their embedded ptx. When looking at the performance tests it even looks like enabling LDGs in the `gpu_horizontal` backend (which was implemented in `gpu_horizontal/entry_point.hpp`) failed before this PR for whatever reason.
But implementing the binary checks is probably still too much of work. In the end we should detect performance regressions/changes when this fails.